src/mlpack/tests/ann_layer_test.cpp

```diff
@@ -4293,6 +4293,78 @@ TEST_CASE("AdaptiveMeanPoolingTestCase", "[ANNLayerTest]")
   REQUIRE(arma::accu(delta) == 2.25);
 }
 
+/**
+ * Simple Separable Convolution layer test.
+ */
+TEST_CASE("SimpleSeparableConvolutionLayerTest", "[ANNLayerTest]")
+{
+  arma::mat input(224 * 224 * 3, 1), output;
```

Comment:
Lets also add test cases similar to convolution layer i.e. by varying values of parameters (especially num_groups here).
Also we should add a test to check value for backward function. 
Ref : https://github.com/mlpack/mlpack/blob/42071e92be63221da73a620de5beac0a5c537a9c/src/mlpack/tests/ann_layer_test.cpp#L2438

Reply:
Hi, will add both the tests but do you think it's a better idea to identify the problem i have first because i was not able to find any solution to that and haivng mismatching values when i run them locally vs when on pipeline is creating a lot of problems.

Reply:
Hey @kartikdutt18 I am not sure how to get the backward value on a module in pytroch, do you have something i can look into for this? 
cc: @zoq